test/e2e/page-objects/my-tickets.js

```diff
@@ -0,0 +1,66 @@
+const BasePage = require('./base-page');
+
+const MyTicketPage = Object.create(BasePage, {
+  open: {
+    value() {
+      return BasePage.open.call(this, '/dashboard/tickets');
+    },
+  },
+  events: {
+    get() {
+      return $$('.cd-user-ticket-list-item.cd-event-ticket-list__event');
+    },
+  },
+  eventName: {
+    value(index) {
```

Comment:
I know this pattern is copied from elsewhere, but I really dislike the passing of index in here. To me it makes more sense to just have a get and return the list, then access the specific index by a normal `[index]` from the test instead. Only case where it makes sense to pass in an index is when the selector itself is selecting by index, rather than grabbing an item at a position from a list.

Reply:
So you would rather pass down to EventName its parent path and get the name from there?

Reply:
No, I mean it should be just
```
eventNames: {
  get() {
    return $('.cd-user-ticket-list-item.cd-event-ticket-list__event .cd-user-ticket-list-item__name');
  },
},
```

Then, when wanting to get the _n_th eventName, you just go `MyTicketsPage.eventNames[n]`

Reply:
(It's no biggie, and something we could do as a sweeping refactor instead, since this is far from the only offender)

Reply:
I know but this does only work for eventName (a "one to one" relationship). If we try for tickets (a "one to n") for example, it becomes a mess because you have to count the number of overall tickets.
That's why i'd think using the event "as a root" reference to the css selector is better